configs/audit.yml

```
@@ -1,5 +1,7 @@
 name: "GitHub Community Pack Audit Configuration"
 
+threat-models: local
```

`threat-models` is an array
```suggestion
threat-models:
  - local
```

---

Ignore, it can be either